### PR TITLE
fix: editor contains previous comment when reloading page after successful comment to a proposal

### LINKED ISSUE #876: Comment draft not cleared when reloading page after successful comment to a proposal
**Affected Portal:**
Select one: All

**Description:**
When commenting to a proposal, the comment editor is cleared, and the comment appears above it, just as is should. However, if you reload the page, the previous comment draft is back in the comment editor again.

**User Type:**
Specify: Regular User

**Reproduction Steps**

The issue is reproduced in the Playwright test in PR #875 

Here is a video reproducing the issue, notice that the comment comes back into the editor in the end:

https://github.com/user-attachments/assets/c1ac73a9-2389-4ae8-b869-ac9cd8cf4a60

**Expected behavior**
The comment editor should be cleared, also after reloading the page.

**Visual Reference**
If applicable, add screenshots or video recording to help explain your problem.

**Priority**
Select One: Medium (P2) – Major inconvenience

```[tasklist]
### Tasks
- [x] Reproduce issue in Playwright test
- [x] Implement fix for test
- [x] Apply fix and test in infrastructure-committee
- [x] Apply fix and test in events-committee ( depends on #843 )
```

### PR DESCRIPTION
**Pull request by petersalomonsen**
- test for large proposal comment
- test that the submitted transaction contains the pasted comment text
- test that the comment appears in the log
- test that the comment editor field is cleared after successful submission
- test for reloading the page, check if the comment editor field is still empty ( which it is not )
- fix implemented for devhub.near
- fix implemented for events-committe.near by copying in the affected files as is, and running the proposal comment test suite in the workflow. Consider symlinking identical component files in the future
- fix implemented for infrastructure-committee.near by applying the relevant changes into infrastructure component sources. The sources have diverged too much from devhub, so not possible to just use the devhub components as is. The same test suite for proposal comments are also applied to IC.

https://github.com/user-attachments/assets/798bad82-0b0b-4e80-b617-92e95af52f06

https://github.com/user-attachments/assets/59899349-7e4f-4ff5-814d-ef84d8abce95

https://github.com/user-attachments/assets/c24c7a82-61b9-46d3-8629-e80a83482025

fixes #876

### CODE REVIEW

**Comment by petersalomonsen** [comment]
@race-of-sloths please, include my PR in the Race

**Comment by race-of-sloths** [comment]
@petersalomonsen Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
New Sloth joined the Race! Welcome!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/petersalomonsen?type=bot&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/petersalomonsen?type=bot&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/petersalomonsen)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/petersalomonsen)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Comment by Tguntenaar** [comment]
@race-of-sloths score 3

**Comment by race-of-sloths** [comment]
### 🌟 Score recorded!

@Tguntenaar, thank you for scoring this pull request in the Race of Sloths!

**Review by Tguntenaar — APPROVED** [review]
LGTM

**Comment by race-of-sloths** [comment]
### ✅ PR is finalized! 
Your contribution is much appreciated with a final score of 3!
You have received 40 (30 base + 10 weekly bonus) Sloth points for this contribution


Another weekly streak completed, well done @petersalomonsen! To keep your weekly streak and get another bonus make pull request next week! Looking forward to see you in race-of-sloths

File: playwright.config.js
Comment (Tguntenaar) [inline playwright.config.js:72]: @petersalomonsen this change is not in the CI only locally. These tests don't work for the `events-commitee.near` account, that is why I added `events/proposals.spec.js`.

Comment (petersalomonsen) [reply playwright.config.js:72]: good point, I did separate out the proposal comment test and added a workflow task for that as you can see here: https://github.com/NEAR-DevHub/neardevhub-bos/blob/main/.github/workflows/continuous-integration-workflow-events.yml#L22, but I forgot to set it for the whole proposal directory which I believe also will work. Will do so in the next PR.

**Pull request merged**
